doc/snek.adoc

```diff
@@ -2415,8 +2459,8 @@ get the board into this mode by connecting the board to your computer
 over USB and then pressing the reset button twice in quick
 succession.
 
-Then, find the snek-feather-{version}.uf2 file included in the snek
-package for your machine and copy it to the CURRENT.UF2 file on the
+Then, find the `snek-feather-`{version}`.uf2` file included in the Snek
```

Comment:
Hrm. {version} should be set in the same style as snek-feather here.

Reply:
For all of these I think it's fine. I didn't want to work out how to fix the formatting, and it doesn't really matter.